feature-libs/checkout/base/components/checkout-delivery-mode/checkout-delivery-mode.component.html

```diff
@@ -41,26 +41,22 @@ <h2 class="cx-checkout-title d-none d-lg-block d-xl-block">
     </div>
   </div>
 
-  <ng-container *ngIf="!continueButtonPressed; else loading">
-    <div class="row cx-checkout-btns">
-      <div class="col-md-12 col-lg-6">
-        <button class="btn btn-block btn-action" (click)="back()">
-          {{ backBtnText | cxTranslate }}
-        </button>
-      </div>
-      <div class="col-md-12 col-lg-6">
-        <button
-          class="btn btn-block btn-primary"
-          [disabled]="
-            deliveryModeInvalid || (changeDeliveryModeInProgress$ | async)
-          "
-          (click)="next()"
-        >
-          {{ 'common.continue' | cxTranslate }}
-        </button>
-      </div>
+  <div class="row cx-checkout-btns">
+    <div class="col-md-12 col-lg-6">
+      <button class="btn btn-block btn-action" (click)="back()">
```

Comment:
Should we also disable the _back_ button if busy$?

Reply:
I think it's fine like they're not moving to the next step, but going back. What do you think?